pkg/inference/backends/llamacpp/llamacpp_config.go

```diff
@@ -20,8 +20,8 @@ type Config struct {
 func NewDefaultLlamaCppConfig() *Config {
 	args := append([]string{"--jinja", "-ngl", "999", "--metrics"})
 
-	// Special case for Windows ARM64
-	if runtime.GOOS == "windows" && runtime.GOARCH == "arm64" {
+	// Special case for ARM64 (aarch64 on Linux)
+	if runtime.GOARCH == "aarch64" || runtime.GOARCH == "arm64" {
```

Comment:
Moreover, this would be `true` on macOS as well.

Reply:
Yup, which is actually fine. Threads is irrelevant on macOS as we use the GPU for inferencing.